edgelessdb/getting-started/quickstart-azure.md

```diff
@@ -0,0 +1,91 @@
+# Quickstart: Microsoft Azure
+In this guide, you will set up EdgelessDB using the Microsoft Azure Marketplace.
+
+## Get the machine
+Visit the Azure Marketplace and get [EdglessDB](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/edgelesssystems.edb?tab=Overview).
+
+![azure_create](../_assets/azure/azure_get.png)
+
+After that you have to configure the virtual machine in the creation wizard.
+
+!> Important: To experience the full potential of EDB we recommend to choose a virtual machine size of the [DCsv2-series](https://docs.microsoft.com/en-us/azure/virtual-machines/dcv2-series). This guarantees, that your virtual machine supports Intel SGX.
+
+The resource group you deploy your machine to has to be created in one of the regions where the DCsV2 sizes are [available](https://azure.microsoft.com/de-de/global-infrastructure/services/?products=virtual-machines&regions=all).
+
+The value for the `Virtual machine name` can be freely selected.
+The value for the `Username` **must** be set to _edb_.
+Finally press the blue button `Review + Create` and after that the `Create` button to finish the setup process.
+
+![azure_wizard](../_assets/azure/azure_wizard_edited.png)
+
+## Get the IP
+After you have created the machine you need the public IP address to interact with it.
+For that naviagte to the resource group your virtual machine was created in and select the virtual machine.
+The public IP address is displayed on the right hand side in the `Essentials` section.
+
+Alternatively you can use the [azure-cli](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) to get the public IP address:
+```bash
+ az vm show -d -g <resource-group> -n <vm-name> --query publicIps -o tsv
+```
+
+## Generate certificates and create a manifest
+You will now create a manifest that defines a root user. This user is authenticated by an X.509 certificate.
+
+Generate a certificate authority (CA) and a corresponding user certificate:
+```bash
+openssl req -x509 -newkey rsa -nodes -subj '/CN=My CA' -keyout ca-key.pem -out ca-cert.pem
+openssl req -newkey rsa -nodes -subj '/CN=rootuser' -keyout key.pem -out csr.pem
+openssl x509 -req -CA ca-cert.pem -CAkey ca-key.pem -CAcreateserial -in csr.pem -out cert.pem
+```
+
+Escape the line breaks of the CA certificate:
+```bash
+awk 1 ORS='\\n' ca-cert.pem
+```
+
+Create a file `manifest.json` with the following contents:
```

Comment:
```suggestion
Create a `manifest.json` file with the following content:
```

Reply:
AFAIK contents is correct here.
Regarding "file manifest.json" vs "manifest.json file": Not sure, but to me the former is short for "a file named manifest.json" and the latter would emphasize the kind of file (e.g., system file, JPEG file), but we would probably want to emphasize the name of the file.